source/vibe/http/websockets.d

```diff
@@ -418,7 +425,8 @@ final class WebSocket {
 		Sends a message using an output stream.
 		Throws: WebSocketException if the connection is closed.
 	*/
-	void send(scope void delegate(scope OutgoingWebSocketMessage) sender, FrameOpcode frameOpcode = FrameOpcode.text)
```

Comment:
There should be an additional compatibility wrapper here to avoid breaking code:

    /// Compatibility overload - will be removed soon.
    deprecated("Need to specify an explicit FrameOpcode.")
    void send(scope void delegate(scope OutgoingWebSocketMessage) sender)
    {
        send(sender, FrameOpcode.text);
    }